teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -131,194 +134,110 @@ surfacing the verification information and building a corpus of verified resourc
-the Tekton Hub, as long as they comply with the Catalog contract. The
-Tekton Hub can indicate the source Catalog of each resource, such as
-through a badge or tag. 
-
-To add the Official Catalog, we only have to modify the Catalogs 
-[configuration][hub-config] in the Hub, as such:
+##### Additions to Verified Catalogs
 
-```yaml
-catalogs:
-  
-  # community catalog
-  - name: tekton-community
-    org: tektoncd
-    type: community
-    provider: github
-    url: https://github.com/tektoncd/catalog
-    revision: main
-    
-  # official catalog  
-  - name: tekton-official
-    org: tektoncd
-    type: official
-    provider: github
-    url: https://github.com/tektoncd/catalog-official
-    revision: main
-```
+Adding a new Verified Catalog or adding a resource into an Verified Catalog should be an exception, not the norm. The 
+set of Verified Catalogs and their resources should be relatively small so that it is sustainable to maintain them. 
+
+A new Verified Catalog may be created, or a new resource added to it if:
+* _Quality_: It meets the [requirements of the Verified Catalog](#requirements-1).
+* _Bandwidth_: The [Tekton Catalog Maintainers][catalog-owners] approve its addition based on its usefulness to the
+community and existing bandwidth to maintain the resource.
 
-This provides extensibility to the support tiers: we can add additional
-Catalogs for other support tiers as needed in the future. 
+#### Hub
 
-#### Community and Official Catalogs in the Tekton CLI
+Users rely on the [Artifact Hub][hub] to discover shared resources. The Artifact Hub supports publishing resources from
+multiple Catalogs. Users and organizations can create their own Catalogs and share them in the Artifact Hub ([guide](https://artifacthub.io/docs/topics/repositories/tekton-tasks/)), as long as they comply with the Catalog contract.
 
-Today, users can use the CLI to install resources from Catalogs by
-passing in the Catalog name to the `--from` argument as shown in the
-examples below. This will add a label to the resource indicating the
-source Catalog: `hub.tekton.dev/catalog=<catalog-name>`.
+To distinguish the Catalogs support tier in the Hub, the [Tekton Catalog Maintainers][catalog-owners] has reserved the `tektoncd` org in the Artifact Hub. **Only** the catalogs published under the `tektoncd` org are with the `Verified` support tier. The community contributors are able to create their own Artifact Hub organization to publish Catalogs, the support tier of such Catalogs are `Community`.
+
+the [Tekton Catalog Maintainers][catalog-owners] also reserved the `tektoncd-legacy` org in the Hub, surfacing the current [centralized Catalog repo](https://github.com/tektoncd/catalog) following the legacy directory-based versioning. We will stop surfacing catalogs from the centralized Catalog repo in the Artifact Hub after the 9-month migration period as discussed in [TEP-0115][tep-0115-migration].
+
+#### CLI
```

Comment:
@dibyom I addressed the comments you made in https://github.com/tektoncd/community/pull/793 in the `CLI` and `Cluster` section, PTAL 😁